reference/6/Microsoft.PowerShell.Management/Start-Process.md

```diff
@@ -396,6 +396,8 @@ Otherwise, this cmdlet does not return any output.
 ## NOTES
 * This cmdlet is implemented by using the **Start** method of the **System.Diagnostics.Process** class. For more information about this method, see [Process.Start Method](https://msdn.microsoft.com/library/system.diagnostics.process.start) in the MSDN library.
 
+* When using PowerShell Core on Linux, to open a new process within a new window (similar to the default behavior when using Start-Process in Windows), run the cmdlet with the `-UseNewEnvironment -Wait` parameters.  This is done to prevent the new process from blocking PowerShell's control of keyboard input.
```

Comment:
Thanks for taking this on, @1RedOne, but this information is not correct:

* Creating a new _window_ is fundamentally unsupported on Unix-like platforms (macOS and Linux).
   * As an aside: Please either mention both macOS and Linux explicitly, if both platforms are affected or use a term such as _Unix-like platforms_; @sdwheeler, is there established terminology?

* `-Wait` is only needed if you're launching a terminal-based (non-GUI) utility (to prevent the keyboard / output issue), as opposed to asynchronously launching a GUI, such as `gedit`, which is the other legitimate use case.

   * However, the only reason to run a terminal-based utility with `Start-Process -Wait` is in the _unusual_ event that you want to ignore the current process' environment and start with a fresh one via `-UseNewEnvironment` - without that, the utility can just be invoked _directly_ (which makes the invocation _implicitly_ synchronous and also keeps the new process' standard streams connected to PowerShell's).

In short, on Unix-like platforms:

* `-NoNewWindow` is _invariably implied_ - you'll never get a new terminal window.
* Use `Start-Process` _only_ in these scenarios:
   * To launch a GUI application that has a blocking CLI (e.g., `gedit`, `firefox`) asynchronously - typical only on Linux.
   * To launch a terminal-based utility with `-UseNewEnvironment` to start with a fresh environment, which then also requires use of `-Wait` to prevent the keyboard-control issues and/or preventing output from arriving asynchronously. With `-Wait`, output will print synchronously to the _terminal_, but due to the process not being connected to PowerShell's streams you won't able to capture or suppress it - which is the price of using `Start-Process`.